### ISSUE TITLE
smarthome.config.xml.test failure: NoClassDefFound

### ISSUE THREAD

**Issue by 9037568** [eclipse/smarthome#4837]
When attempting to build the package from the project root using `mvn clean install`:

```
assert that ConfigDescriptions were loaded properly(org.eclipse.smarthome.config.xml.test.ConfigDescriptionsTest)  Time elapsed: 0.063 sec  <<< ERROR!
java.lang.NoClassDefFoundError: org/eclipse/smarthome/config/xml/osgi/AbstractAsyncBundleProcessor
        at org.eclipse.smarthome.test.SyntheticBundleInstaller.waitUntilLoadingFinished(SyntheticBundleInstaller.java:235)
        at org.eclipse.smarthome.test.SyntheticBundleInstaller.install(SyntheticBundleInstaller.java:97)
        at org.eclipse.smarthome.test.SyntheticBundleInstaller.install(SyntheticBundleInstaller.java:73)
        at org.eclipse.smarthome.test.SyntheticBundleInstaller$install.call(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:120)
        at org.eclipse.smarthome.config.xml.test.ConfigDescriptionsTest.assert that ConfigDescriptions were loaded properly(ConfigDescriptionsTest.groovy:64)
```

[...]

**Comment by kaikreuzer** [comment]
The current master works smoothly on Jenkins: https://ci.eclipse.org/smarthome/job/SmartHomeDistribution-Nightly/2407/

Can you make out any difference to your setup?

**Comment by 9037568** [comment]
As far as I can tell I'm working off the latest version of the current master branch. Yes, it's forked, but I can see no changes outside of the new binding I'm working on...

I'll kick off a run of a completely clean master and see if there's any difference.

**Comment by 9037568** [comment]
Ah, yes, I remember now.

Starting from a completely clean master, the first failure encountered is #4370.
Working around that leads to the error described in this ticket.

So perhaps this is related to the platform (ie. only occurs on Windows) ?

**Comment by kaikreuzer** [comment]
Now that https://github.com/eclipse/smarthome/pull/4685 is merged, could you try the build without any workaround?

**Comment by 9037568** [comment]
I will do that tonight.

**Comment by 9037568** [comment]
Confirmed that this problem no longer occurs after the fix for #4685.
Closing this accordingly.
